### PR TITLE
Update NetSwitchTeam.md

### PR DESCRIPTION
**Pull request by Deland-Han**
Adding a note for readers. See [CI111820](https://dev.azure.com/ContentIdea/ContentIdea/_queries/edit/111820/?triage=true) for details.

### CODE REVIEW

**Comment by Deland-Han** [comment]
#sign-off

**Comment by e0i** [comment]
@JohanFreelancer9 Minor style checks needed for this PR. Thanks.

**Comment by e0i** [comment]
@dariomws306 @get-itips @tseward

Peer review needed for this PR. Thanks.

File: docset/windows/netswitchteam/NetSwitchTeam.md [hunk 1]
```diff
@@ -27,6 +27,10 @@ This reference provides cmdlet descriptions and syntax for all Network Switch Te
 These Network Switch Team cmdlets should only be used with NIC switch teams that are manipulated by a forwarding extension in the Hyper-V Extensible Switch.
 
 ## NetSwitchTeam Cmdlets
+
+> [!Note]
```
Comment (JohanFreelancer9) [inline docset/windows/netswitchteam/NetSwitchTeam.md]:
```suggestion
> [!NOTE]
```

**Comment by Deland-Han** [comment]
@tseward @JohanFreelancer9 Thank you for reviewing the pull request. Did you have an ETA for merging the pull request?

**Comment by Deland-Han** [comment]
@get-itips and @dariomws306. As you approved the pull request, did you know who can help merge it?

**Comment by dariomws306** [comment]
@Deland-Han, let me check.

Maybe @v-anbarr?

**Comment by tiburd** [comment]
Hi Deland/@Deland-Han,

We're closing this PR because merging to Live is restricted to the PubOps team only. To find out more about the process, see the [PubOps Publishing Process page](https://review.docs.microsoft.com/en-us/office-authoring-guide/pubops-publishing-process?branch=master). If you resubmit this to the Master branch, the PubOps team will review, merge when ready, and publish to Live. 

Thanks!

**Comment by e0i** [comment]
@tiburd Recreated the same PR targeting the master branch: https://github.com/MicrosoftDocs/windows-powershell-docs/pull/1671

**Pull request closed without merging**